docs/quick-start.md

```diff
@@ -76,6 +76,27 @@ function yy {
     }
     Remove-Item -Path $tmp
 }
+```
+
+  </TabItem>
+  <TabItem value="command-prompt" label="Command Prompt">
+
+Create the file `yy.cmd` and place it in your `%PATH%`.
+
+```cmd
+@echo off
+
+set tmpfile=%TEMP%\tempfile.txt
```

Comment:
Ah sorry I didn't make it clear - is it possible to use a random filename instead of the hardcoded `tempfile.txt` here?

Reply:
> Ah sorry I didn't make it clear - is it possible to use a random filename instead of the hardcoded `tempfile.txt` here?

sure! `%random%` will generate a random number between 0 and 32767

Reply:
That's neat, thanks for the change!